Runtime/Interaction/StaticMesh/HPUIStaticContinuousInteractable.cs

```diff
@@ -0,0 +1,28 @@
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine;
+
+namespace ubco.ovilab.HPUI.StaticMesh
```

Comment:
Lets keep the namespace flat for now. Make this "ubco.ovilab.HPUI.Interaction" for now. We'll have a separate namespace if we need to down the line.

Reply:
fixed 🫡